.storybook/preview.tsx

```diff
@@ -39,17 +41,23 @@ const translations = Object.fromEntries(
   }),
 );
 
+const l10n = { ...defaultTranslations, ...translations };
```

Comment:
I guess `translations` is now obsolete? Won't it be the same as `defaultTranslations`, and just for mock data anyways?

```suggestion
const l10n = defaultTranslations;
```